### PR TITLE
Gateway in whats new

### PR DESCRIPTION
**Pull request by jburel**
Add section about gateway.

### CODE REVIEW

File: omero/developers/whatsnew.txt [hunk 1]
```diff
@@ -41,3 +40,9 @@ OMERO developers who have not migrated yet should refer to
 :doc:`Server/GraphsMigration` and take action before the deprecated features
 are removed.
 
+Java Gateway
+^^^^^^^^^^^^
+The Java gateway has undergone a major overhaul.
```
Comment (hflynn) [inline omero/developers/whatsnew.txt]: You need a blank line under the ^^^^ before you start the text

File: omero/developers/whatsnew.txt [hunk 2]
```diff
@@ -41,3 +40,9 @@ OMERO developers who have not migrated yet should refer to
 :doc:`Server/GraphsMigration` and take action before the deprecated features
 are removed.
 
+Java Gateway
+^^^^^^^^^^^^
+The Java gateway has undergone a major overhaul. 
+This should make the development of Java applications much easier. See
+:doc:`/developers/Java` for more information or follow the example in
+  `minimal-omero-client <https://github.com/ome/minimal-omero-client>`_.
```
Comment (hflynn) [inline omero/developers/whatsnew.txt]: This last line is indented which is why Travis is failing

**Comment by hflynn** [comment]
Reads fine, should be good to merge once the two formatting issues are fixed and Travis is happy.

**Comment by jburel** [comment]
@hflynn: travis is green now

**Comment by hflynn** [comment]
Looks good on staging thanks, merging.

**Comment by joshmoore** [comment]
@jburel: is there already a location to specify the API guarantees for Gateway? i.e. "breaking changes will be handled by ..."

**Pull request merged**